tests/components/jewish_calendar/test_binary_sensor.py

```diff
@@ -86,7 +173,9 @@ async def test_issur_melacha_sensor(
 
     registry = await hass.helpers.entity_registry.async_get_registry()
 
-    with alter_time(test_time):
+    with alter_time(test_time), patch(
+        "homeassistant.helpers.event.async_track_point_in_time"
```

Comment:
Don't patch this. Move time forward and check that state is updated as expected.

Reply:
Can you please elaborate? We are setting the time to a past date (in 2018). What do you mean "move time forward"?

Reply:
Done